### PR TITLE
Refactored download command to check MD5 checksums set in env.sh

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

**Comment by mikewalch** [comment]
This pull request removes calls to `gpg verify` in `fluo-dev download` and should close #66

**Comment by drewfarris** [comment]
ship it!

File: bin/impl/download.sh [hunk 1]
```diff
@@ -36,8 +58,13 @@ if [ -n "$ACCUMULO_TARBALL_REPO" ]; then
   popd
   cp $ACCUMULO_BUILT_TAR $DOWNLOADS/
 else
-  ACCUMULO_PATH=accumulo/$ACCUMULO_VERSION
-  wget -c -P $DOWNLOADS $APACHE_MIRROR/$ACCUMULO_PATH/$ACCUMULO_TARBALL
+  download_verify $APACHE_MIRROR/accumulo/$ACCUMULO_VERSION $ACCUMULO_TARBALL $ACCUMULO_MD5
+fi
+
+if [ -z "$FLUO_TARBALL_PATH" -a -z "$FLUO_TARBALL_REPO" -a -n "$FLUO_TARBALL_URL" ]; then
+  FLUO_TARBALL=fluo-distribution-$FLUO_VERSION-bin.tar.gz
```
Comment (keith-turner) [inline bin/impl/download.sh]: why not call download_verify here?

File: bin/impl/redeploy.sh [hunk 2]
```diff
@@ -15,7 +15,18 @@
 # limitations under the License.
 
 # stop if any command fails
-set -e  
+set -e
+
+function verify_checksum() {
+  tarball=$1
+  expected_md5=$2
```
Comment (keith-turner) [inline bin/impl/redeploy.sh]: could have a sanity check on expected_md5... ensure its of correct len and has expected chars....

Comment (keith-turner) [inline bin/impl/redeploy.sh]: my thinking is that the sanity check on expected_md5 makes the equality check below a bit more robust

**Comment by keith-turner** [comment]
+1

File: bin/impl/redeploy.sh [hunk 3]
```diff
@@ -35,21 +35,20 @@ elif [ -n "$FLUO_TARBALL_REPO" ]; then
...
     echo "The tarball $TARBALL does not exist after building from the FLUO_TARBALL_REPO=$FLUO_TARBALL_REPO"
     echo "Does your repo contain code matching the FLUO_VERSION=$FLUO_VERSION set in env.sh?"
     exit 1
   fi
-elif [ -n "$FLUO_TARBALL_URL" ]; then
-  TARBALL_FILENAME=fluo-distribution-$FLUO_VERSION-bin.tar.gz
-  if [ -f "$DOWNLOADS/$TARBALL_FILENAME" ]; then
-    echo "$TARBALL_FILENAME already exists in downloads/"
+elif [ -n "$FLUO_TARBALL_URL_PREFIX" ]; then
+  if [ -f "$DOWNLOADS/$FLUO_TARBALL" ]; then
```
Comment (keith-turner) [inline bin/impl/redeploy.sh]: Should redeploy.sh download?    Thinking it would be more consistent if only download.sh did that.

**Pull request merged**